images/Dockerfile.epics

```diff
@@ -1,17 +1,17 @@
 ARG BASE_IMAGE_TAG
-ARG ANSIBLE_TAG
 FROM dockerregistry.lnls-sirius.com.br/fac/fac-base:${BASE_IMAGE_TAG}
 
 SHELL ["bash", "-c"]
 
 WORKDIR /opt
 
+ARG ANSIBLE_TAG
```

Comment:
I forgot to say that every time the command `FROM` is called, the `ARG` is reset.
You did the right thing moving it there. In case you want to use it before and after a from command, the ARG must be declared again. e.g.:

```dockerfile
ARG ANSIBLE_TAG
# ANSIBLE_TAG is available .....
...
FROM dockerregistry.lnls-sirius.com.br/fac/fac-base:${BASE_IMAGE_TAG}
# ANSIBLE_TAG is empty
...
ARG ANSIBLE_TAG
# ANSIBLE_TAG is available
...
```

Reply:
@carneirofc , can you approve this PR ?

Reply:
Sure!